src/main/java/com/kodilla/ecommercee/ProductController.java

```diff
@@ -1,4 +1,24 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/product")
 public class ProductController {
+
+    @GetMapping
+    public void get(){}
+
+    @GetMapping({"/{id}"})
+    public void get(@PathVariable Long id){}
+
+    @PostMapping
+    public void create(){}
```

Comment:
create cos powinno dostac ;) create(ProductDto) i cos zwrocic (tez moze byc ProductDto)

Reply:
No tak, wiem o tym, ale nie mam stworzonych jeszcze takich encji.

Reply:
to nie encja :) Mówiłem, aby stworzyć pustą klasę (w pakiecie dto) ProductDto (bez zadnych pol, metod, konstruktora)